### PR TITLE
Preparation for release 1210.12.107.1000

### PR DESCRIPTION
**Pull request by NimishaS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +49,11 @@ It is an ordered list of roles and/or recipes that are run in the exact order de
 
 `validation_key_format`: Specifies the format in which `validation_key` is set in the `privateconfig.config` file. Supported values are `plaintext` and `base64encoded`. Default value is `plaintext`.
 
-`bootstrap_version`: Set the version of `chef-client` that needs to get installed on the VM. This option is supported only for linux extension.
+`bootstrap_version`: Set the version of `chef-client` that needs to get installed on the VM.
+
+`daemon`: Supported only for windows node. Configures the chef-client service for unattended execution. Supported values are `auto` and `service`. Default is `service`.
+`auto` uses the default behavior. Currently prevents the chef-client service from being configured as a service.
```
Comment (btm) [inline README.md]: It is a little confusing that 'service' is the default, but auto uses the default behavior which is to do nothing? Those seem contrary, we should word them so they aren't.

File: README.md [hunk 2]
```diff
@@ -48,7 +49,11 @@ It is an ordered list of roles and/or recipes that are run in the exact order de
 
 `validation_key_format`: Specifies the format in which `validation_key` is set in the `privateconfig.config` file. Supported values are `plaintext` and `base64encoded`. Default value is `plaintext`.
 
-`bootstrap_version`: Set the version of `chef-client` that needs to get installed on the VM. This option is supported only for linux extension.
+`bootstrap_version`: Set the version of `chef-client` that needs to get installed on the VM.
+
+`daemon`: Supported only for windows node. Configures the chef-client service for unattended execution. Supported values are `auto` and `service`. Default is `service`.
```
Comment (btm) [inline README.md]: capitalize Windows

File: README.md [hunk 3]
```diff
@@ -48,7 +49,11 @@ It is an ordered list of roles and/or recipes that are run in the exact order de
 
 `validation_key_format`: Specifies the format in which `validation_key` is set in the `privateconfig.config` file. Supported values are `plaintext` and `base64encoded`. Default value is `plaintext`.
 
-`bootstrap_version`: Set the version of `chef-client` that needs to get installed on the VM. This option is supported only for linux extension.
+`bootstrap_version`: Set the version of `chef-client` that needs to get installed on the VM.
```
Comment (btm) [inline README.md]: Set the version of `chef-client` to be installed on the VM.

File: README.md (hunk 1 above)
Comment (NimishaS) [inline README.md]: Should I just keep it like this?
```
`auto` Currently prevents the chef-client service from being configured as a service.
```
The same needs to be modified here too: https://github.com/chef/knife-azure/pull/417/files#diff-f3b483eb11d334acbcfadd2d19f190bbR97
@btm , we had previously kept the supported values for `-daemon` as `service/task/none`, but we were suggested to replace `none` by `auto`: https://github.com/chef/mixlib-install/pull/167#discussion_r87705386

**Pull request merged**